### PR TITLE
Add Comment Write APIs to docs

### PR DESCRIPTION
**Pull request by nimeshnayaju**
### Description

- [x] Create thread
- [x] Edit thread metadata
- [x] Create comment
- [x] Edit comment
- [x] Delete comment
- [x] Add comment reaction
- [x] Remove comment reaction

### CODE REVIEW

File: docs/references/v2.openapi.json [hunk 1]
```diff
@@ -1221,18 +1222,113 @@
...
-          },
           "404": {
-            "$ref": "#/components/responses/404"
+            "$ref": "#/components/responses/404-thread"
+          }
+        }
+      },
+      "post": {
+        "summary": "Create thread",
+        "description": "This endpoint creates a new thread.",
```
[the remark this answers is not in the document]
Comment (CTNicholas) [reply docs/references/v2.openapi.json:1232]: I think we need more info on how to create a `CommentBody`. I think we need a proper guide, but for now you could add this:

> `content.body` is an array of paragraphs, each containing text nodes, for example:
> ```json
> [
>  {
>    "type": "paragraph",
>    "children": [{ "text": "Hello " }, { "text": "world", "bold": true }]
>  }
> ]
> ```
> Note that `content` corresponds to the `CommentBody` type exported from `@liveblocks/node`.

File: docs/pages/api-reference/liveblocks-node.mdx [hunk 2]
```diff
@@ -1006,6 +1076,86 @@ const comment = await liveblocks.getComment({
...
+    },
+    userId: "user1",
+    editedAt: new Date(), // Optional, the date and time of the edit
+  },
+});
+```
+
+#### Liveblocks.deleteComment [#delete-comment]
+
+Deletes a specific comment from a thread within a room. Throws an error if the
```
Comment (FlowFlorent) [inline docs/pages/api-reference/liveblocks-node.mdx:1106]:
```suggestion
Deletes a specific comment from a thread within a room. If there are no remaining comments in the thread, the thread is also deleted. Throws an error if the
```

File: docs/references/v2.openapi.json [hunk 3]
```diff
@@ -1221,18 +1222,113 @@
               }
             }
           },
-          "401": {
-            "$ref": "#/components/responses/401"
-          },
-          "403": {
```
[the remark this answers is not in the document]
Comment (nimeshnayaju) [reply docs/references/v2.openapi.json:1227]: Oops. Totally forgot about the middlewares 🫢

File: docs/references/v2.openapi.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (nimeshnayaju) [reply docs/references/v2.openapi.json:1238]: Thank you! Added this blurb to the `@liveblocks/node` section of our documentation:

https://github.com/liveblocks/liveblocks/blob/15dcf044299b0ef593fc64ca612ed1f82fa27839/docs/pages/api-reference/liveblocks-node.mdx#L1042-L1057

**Comment by CTNicholas** [comment]
I'll finish up tomorrow 👍 great work on starting this!

**Review by CTNicholas — COMMENTED** [review]
I've made a ton of changes, I think it's just about there. I'd appreciate it if @FlowFlorent or @nimeshnayaju could quickly check through what I've written!

Might need to make a change for this, but otherwise I'm happy: 
- https://github.com/liveblocks/liveblocks/pull/1325#issuecomment-1842864386

**Comment by nimeshnayaju** [comment]
Thanks for making this PR so much better @CTNicholas . Going through the changes right now, and I think it is very well written and explained. Thanks for the amazing work! ❤️

**Review by FlowFlorent — APPROVED** [review]

File: docs/pages/api-reference/liveblocks-node.mdx [hunk 4]
```diff
@@ -947,62 +947,473 @@ console.log(schema);
...
+    userId: "guillaume@example.com",
+    createdAt: new Date(), // Optional, the time the reaction was added
+  },
+});
+
+// { emoji: "👨‍👩‍👧", userId "guillaume@example.com", ... }
+console.log(reaction);
+```
+
+#### Liveblocks.deleteCommentReaction [#post-rooms-roomId-threads-threadId-comments-commentId-remove-reaction] [@badge=Beta]
```
Comment (FlowFlorent) [inline docs/pages/api-reference/liveblocks-node.mdx:1399]:
```suggestion
#### Liveblocks.removeCommentReaction [#post-rooms-roomId-threads-threadId-comments-commentId-remove-reaction] [@badge=Beta]
```

**Review by CTNicholas — APPROVED** [review]

File: docs/pages/api-reference/liveblocks-node.mdx (hunk 4 above)
Comment (nimeshnayaju) [reply docs/pages/api-reference/liveblocks-node.mdx]: Yes, renaming this to `removeCommentReaction`

File: docs/pages/api-reference/liveblocks-node.mdx [hunk 5]
```diff
@@ -947,62 +947,473 @@ console.log(schema);
...
+    // Optional, the time the comment was edited
+    editedAt: new Date(),
+  },
+});
+
+// { type: "comment", threadId: "th_d75sF3...", ... }
+console.log(editedComment);
+```
+
+#### Liveblocks.deleteComment [#post-rooms-roomId-threads-threadId-comments-commentId] [@badge=Beta]
```
Comment (CTNicholas) [inline docs/pages/api-reference/liveblocks-node.mdx:1358]:
```suggestion
#### Liveblocks.deleteComment [#delete-rooms-roomId-threads-threadId-comments-commentId] [@badge=Beta]
```

**Pull request merged**